From 01402a66bb11eab8068571a98fef6753f9b21b9a Mon Sep 17 00:00:00 2001
From: "Dr. Greg" <greg@enjellic.com>
Date: Fri, 5 Feb 2021 04:32:26 -0600
Subject: [PATCH 07/50] Rename event pseudo-files.

This update renames the pseudo-directories that are used to
manage event namespaces.

The following pseudo-directory contains the pseudo-files that
allow interpretation and configuration of an event namespace:

/sys/kernel/security/integrity/events

The following pseudo-directory will contain pseudo-files that
represent event namespaces that are created:

/sys/fs/integrity-events
---
 security/integrity/ima/ima_events.c | 5 +++--
 security/integrity/ima/ima_fs.c     | 2 +-
 2 files changed, 4 insertions(+), 3 deletions(-)

diff --git a/security/integrity/ima/ima_events.c b/security/integrity/ima/ima_events.c
index f4e179ec8fcf..c1aba0e8c17d 100644
--- a/security/integrity/ima/ima_events.c
+++ b/security/integrity/ima/ima_events.c
@@ -72,7 +72,7 @@ static _Bool have_aggregate;
 static char aggregate[WP256_DIGEST_SIZE];
 
 /*
- * Root dentry for the 'iso-identity' directory and dentries for the
+ * Root dentry for the 'events' directory and dentries for the
  * pseudo-files in that directory.
  */
 static struct dentry *identity_dir;
@@ -1714,7 +1714,8 @@ int __init ima_events_init(void)
 	if (IS_ERR(trajectory))
 		goto err;
 
-	ima_events_kobject = kobject_create_and_add("iso-identity", fs_kobj);
+	ima_events_kobject = kobject_create_and_add("integrity-events",
+						    fs_kobj);
 	if (!ima_events_kobject)
 		goto err;
 
diff --git a/security/integrity/ima/ima_fs.c b/security/integrity/ima/ima_fs.c
index 7f43b745d2ad..fdd1a4d18b22 100644
--- a/security/integrity/ima/ima_fs.c
+++ b/security/integrity/ima/ima_fs.c
@@ -453,7 +453,7 @@ struct dentry * __init ima_fs_init_identity(void)
 {
 	struct dentry *id_dir = NULL;
 
-	id_dir = securityfs_create_dir("iso-identity", ima_dir);
+	id_dir = securityfs_create_dir("events", integrity_dir);
 	if (IS_ERR(id_dir))
 		return NULL;
 	return id_dir;
-- 
2.31.1

